### DISCUSSION TITLE
Add a 'polish' category to ratings

### DISCUSSION THREAD

**Discussion by bploeckelman** [ludumdare/ludumdare#2200] [Ideas]
The polish level of a game covers a lot of the existing rating categories. It would be nice to have a separate category for polish so devs are encouraged to think about UX and the presentation of the entire game as a whole.

**Comment by mikekasprzak** [comment]
I'm interested in this idea. Hmmm. 🤔

**Comment by 100thCoin** [comment]
I feel like "polish" ties in pretty heavily with how I score a game for "overall".

**Reply by mikekasprzak** [reply]
Yes, this is the problem with categories. If two categories always yield near identical scores, one of them may not need to exist.

Categories like "Technical" were dropped a long time ago in favor of subjective categories. A subjective category is one that asks for an opinion, which everyone has. Technical ran into the problem that it required an expertise, and many didn't feel qualified.

A question might be: how many games can you find that would score highly I polish, but not other categories? Or is it just an aggregate like Overall?

**Reply by bploeckelman** [reply]
Good points. Treating 'overall' as an aggregate catches a lot of cases and that may be sufficient. 

I've always thought of the winner of the 'overall' category as the winner of the event, and the other categories were indicators of the relative strengths and weaknesses of an entry. That'd likely be an argument _for_ having a 'polish' category.

If people are viewing 'overall' as a combination of the other categories then weaknesses in a certain area would drag down the 'overall' score. If a game just plain wasn't fun or the gameplay was very derivative yet the user experience was really tight and well thought out and there was a lot of juice in each interaction with clear UI and easy to understand gameplay then I'd be likely to rate it higher in a 'polish' category but it would still lose out in 'fun' or 'innovation'.

One other thing that could work against this idea is that there could be a bias towards jammers who've built up a robust set of tools over the course of many jams. The group I've worked with just finished our 26th entry and for the most part know what we can achieve in the allotted time so we can plan for a certain level of cohesiveness (though it doesn't always work out).

Also I imagine adding another category potentially opens a floodgate of "ok then why not this category or this other one" and it may not be worth it.

I'd make a much stronger argument for my other recent suggestion (which I probably should have put in discussion instead of issues) 
https://github.com/JammerCore/JammerCore/issues/2176 

Mike, let me know if you'd like me to move that one out of issues into discussion.

**Reply by mikekasprzak** [reply]
I'm actually open to the idea of new categories, but they need to:

* be subjective - they should not require expertise
* apply to most games - not "best duck" :duck:
* stand on their own

**Reply by 100thCoin** [reply]
Let's take a step back and analyze it from the perspective of giving the developer feedback. What kind of feedback are developers not yet receiving with the current categories? The current categories are open. You want to give me feedback on my Voice Acting, or Sound Design, or Music Compositional Skills, the category for Audio covers all of that in a nice umbrella. The character design is great, decorations and details are fantastic, but the camera is at a poorly chosen angle? Let me know about that in Graphics. What kind of feedback doesn't currently have a category to call home?

I think the existing categories cover all the grounds nicely, but it's a fun thought experiment to brainstorm new categories. The biggest issue is that it's possible for a category to be _too subjective_, like opinions on a game's duration. Other categories like "Replay value" or "Accessibility" would require an opt out, as that's definitely not a high priority for many devs. Even still, adding accessibility options, like a colorblind mode or subtitles, works nicely with the Overall category. And what if your game doesn't need such options? Lots of games don't have voice acting, and black and white minimalist games certainly don't need colorblind modes.

Furthermore, when I'm adding tiny details in my Ludum Dare entries, it's not because I'm trying to get a good score in an arbitrary "details" category. If anything, it might add to Overall, or perhaps Graphics, but we need to consider that the addition of a new category will lead to a lot of people (myself included) trying to focus on that category. What would a game look like if the developer put all their attention on this new proposed category?

Keep in mind, categories like "Humor" aren't for everyone. That's why an option exists to opt out of it. But if we're making a new category, it might as well be something that won't be as optional.

All of this to say I think the existing categories cover everything needed. I would be open to a new category, but I'm just not sure what it would be. Most new categories I can think up tend to be a part of how I rate Overall, and if a new category _is_ added, we need to keep in mind how the developers will consider the new category while making their games, as well as what the new category actually provides the developers in terms of feedback.
